### PR TITLE
fix(client): delete onbording preferences storage

### LINKED ISSUE #173: Clear the local storage of preferences after registration
## The story:
As a user who registered preferences during onbording process, I want to clear the local storage.

---

## What we should do:
We should clear the local storage  so that it doesn't provide previous preferences when the user goes to onbording process again.

---

## To be successful we need to:
_Check these off as they are completed:_
- [ ] Clear the local storage "preferences" after registration.
- [ ] Rename the key name "preferences" to "onbordingPreferences"

### PR DESCRIPTION
**Pull request by TomoyaKuroda**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Fixed issues (PRs without this will not be accepted)
- Fixes #173 

## Changes
- Deleted local storage when user finished his/her onbording process.
- Renamed the key of local storage for onborading

## PR Checklist
- [ ] Does your PR contain relevant tests that pass?
- [ ] Is your code linted? `yarn lint` or `yarn lint:fix`
- [ ] Does your code build correctly in ALL projects? `yarn build`

### CODE REVIEW

File: packages/client/src/pages/Onboarding/Onboarding.page.tsx [hunk 1]
```diff
@@ -127,6 +129,7 @@ export const OnboardingPage = () => {
           onClick={async () => {
             // TODO: Update in Wizard after submission
             await refetch(); // Update the user preferences
+            LocalStorage.onbordingPreferences = null;
```
Comment (tristanMatthias) [inline packages/client/src/pages/Onboarding/Onboarding.page.tsx]: We should probably only do this on a successful submission. Did you want to update that @tomoyakuroda ?

**Review by tristanMatthias — CHANGES_REQUESTED** [review]

File: packages/client/src/pages/Onboarding/Onboarding.page.tsx (hunk 1 above)
Comment (TomoyaKuroda) [reply packages/client/src/pages/Onboarding/Onboarding.page.tsx]: @tristanMatthias  I see. We have updatePreferences mutation in packages\client\src\containers\Wizard.container.ts, so I need to add this code after the mutation, is it correct?

Comment (tristanMatthias) [reply packages/client/src/pages/Onboarding/Onboarding.page.tsx]: Where you have it is fine :). We just want to check if the refetch was _successful_.

EG:
```ts
const res = await refetch();
if (response is a valid req) LocalStorage.onboardingPreferences = null;
else ...
```

Comment (TomoyaKuroda) [reply packages/client/src/pages/Onboarding/Onboarding.page.tsx]: All right. I just added if statement.

**Review by tristanMatthias — APPROVED** [review]

**Pull request merged**